android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/ssdp/SsdpMessage.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2014 Fraunhofer FOKUS
```

Comment:
Also, we will be soon creating an AUTHORS file that will be used for listing companies and then we can insert Fraunhofer FOKUS there.